graph/db/sql_store.go

```diff
@@ -169,12 +171,14 @@ type SQLStore struct {
 	cfg *SQLStoreConfig
 	db  BatchedSQLQueries
 
-	// cacheMu guards all caches (rejectCache and chanCache). If
-	// this mutex will be acquired at the same time as the DB mutex then
-	// the cacheMu MUST be acquired first to prevent deadlock.
-	cacheMu     sync.RWMutex
-	rejectCache *rejectCache
-	chanCache   *channelCache
+	// cacheMu guards all caches (rejectCache, chanCache, and
+	// publicNodeCache). If this mutex will be acquired at the same time as
```

Comment:
you do not need the chacheMu for the LRU cache it is already thread-safe by design and has its own mutex.